canvas_modules/common-canvas/src/common-properties/controls/readonly/readonly.scss

```diff
@@ -62,10 +58,6 @@
 		}
 	}
 	.tooltip-trigger {
```

Comment:
Should this be hear at all?  I would think this would go under the tooltip component.